src/Pack/Runner/Uninstall.idr

```diff
@@ -0,0 +1,26 @@
+module Pack.Runner.Uninstall
+
+import Pack.Config.Types
+import Pack.Core.IO
+import Pack.Core.Logging
+import Pack.Core.Types
+
+%hide Pack.Config.Types.Env.packDir
+
+%default total
+
+--------------------------------------------------------------------------------
+--          Uninstalling Pack
+--------------------------------------------------------------------------------
+
+export covering
+uninstallPack :
+     {auto _ : HasIO io}
+  -> {auto _ : Env}
+  -> EitherT PackErr io ()
+uninstallPack = do
+  info "Uninstalling pack"
+  let msg := "$PACK_DIR: \{packDir}. Continue (yes/*no)?"
```

Comment:
I think, we should add something like "we are going to remote it" to this promt, WDYT?

Reply:
I think it should be fine with the `info` stating "Uninstalling pack", and then subsequently displaying the `$PACK_DIR`, and asking if the user should continue on to uninstall pack with typing "yes"?

Reply:
I think it presupposes the knowledge of how `pack` is installed and works, which is, I think, cannot be expected from anyone calling `uninstall`, that's why I'm asking the message to be a bit more explicit.

Reply:
Ah okay that makes sense, let me address this.

Reply:
This should be addressed via 0484958605bcb12c634a8964fbb81ddc716c8a17.